tests/bugfixes/redmine/test_issue_841.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from system_tests import CaseMeta, path
+
+
+class CrashOnInput(metaclass=CaseMeta):
```

Comment:
[suggestion] Should we rename the test from `crash...` to `throw...` ? The application was crashing before the fix, but now it is throwing an exception.

Reply:
Uh, yeah, now that I re-read the class name, it sounds like it should crash...